docs/configuration.md

```diff
@@ -0,0 +1,198 @@
+# Lighthouse Configuration
+
+The Lighthouse config object is the primary method of customizing a Lighthouse run to suit your use case. Using a custom config, you can limit the audits to run, add additional loads of the page under special conditions, add your own custom checks, tweak the scoring, and more.
+
+## Usage
+
+You can specify a custom config file when using lighthouse through the CLi or consuming the npm module yourself.
+
+**CLI**
+```sh
+lighthouse --config-path=path/to/custom-config.js https://example.com
+```
+
+**Node**
+```js
+const lighthouse = require('lighthouse')
+
+const config = {
+  passes: [{
+    recordTrace: true,
+    pauseBeforeTraceEndMs: 5000,
+    useThrottling: true,
+    gatherers: [],
+  }],
+
+  audits: [
+    'first-meaningful-paint',
+    'speed-index-metric',
+    'estimated-input-latency',
+    'first-interactive',
+    'consistently-interactive',
+  ]
+}
+
+lighthouse('http://example.com/', {port: 9222}, config)
+```
+
+## Properties
+
+### `extends: string|boolean|undefined`
+
+The extends property controls if your configuration should inherit from the default Lighthouse configuration. [Learn more.](#config-extension)
+
+#### Example
+```js
+{
+  extends: 'lighthouse:default',
+}
+```
+
+### `settings: Object|undefined`
+
+The settings property controls various aspects of running Lighthouse such as CPU/network throttling and audit whitelisting/blacklisting.
+
+#### Example
+```js
+{
+  settings: {
+    onlyCategories: ['performance'],
+    onlyAudits: ['works-offline'],
+  }
+}
+```
+
+#### Schema
+| Name | Type | Description |
+| -- | -- | -- |
+| onlyCategories | `string[]` | Limits the run to just what's required for the specified categories. Additive with `onlyAudits`. |
```

Comment:
yeah I was trying to go for something that conveyed the run will be short because passes/audits/gatherers will be filtered not just we don't put the results into the report, but this isn't super fluid, fine with yours if we don't really care about conveying that here